sessiontxn/interface.go

```diff
@@ -100,6 +100,8 @@ const (
 	// AdviceWarmUp indicates to warm up the provider's inner state.
 	// For example, the provider can prefetch tso when it is advised.
 	AdviceWarmUp AdviceType = iota
```

Comment:
I suggest making the comments (and its behavior) explicit and clear, instead of vaguely stating "warm up".

Reply:
done